uvcombine/tests/utils.py

```diff
@@ -109,12 +121,17 @@ def interferometrically_observe_image(image, pixel_scale,
 
     ygrid, xgrid = np.indices(image.shape, dtype='float')
     rr = ((xgrid-image.shape[1]/2)**2+(ygrid-image.shape[0]/2)**2)**0.5
-    rr_uv = (rr / rr.max() / 2. / pixel_scale)
 
     # Create a UV sampling mask.
     # *please sanity check this!*
     # Are the "UV" data correct, or are they off by a factor of 2?
-    ring = (1/rr_uv < largest_angular_scale) & (1/rr_uv > smallest_angular_scale)
+    # rr_uv = (rr / rr.max() / 2. / pixel_scale)
+    # ring = (1/rr_uv < largest_angular_scale) & (1/rr_uv > smallest_angular_scale)
+
+    # EWK -- Something is off in the above masking.
+    img_scale = image.shape[0] * pixel_scale
+    ring = (rr >= (img_scale / largest_angular_scale)) & \
+        (rr <= (img_scale / smallest_angular_scale))
```

Comment:
This is how we've been doing the examples in the other notebooks, though. The two are consistent up to a factor of `2 * max(rr)` (where does that 2 come from?). 

Using the FFT frequencies is consistent with it too:
```
    yfreqs = np.fft.fftshift(np.fft.fftfreq(image.shape[0]))
    xfreqs = np.fft.fftshift(np.fft.fftfreq(image.shape[1]))
    yy_freq, xx_freq = np.meshgrid(yfreqs, xfreqs, indexing='ij')
    freqs = np.sqrt(yy_freq**2 + xx_freq**2)
    ring3 = (freqs > pixel_scale.value / largest_angular_scale.value) & \
        (freqs < pixel_scale.value / smallest_angular_scale.value)
```